service/history/configs/config.go

```diff
@@ -82,6 +82,9 @@ type Config struct {
 	StandbyTaskMissingEventsResendDelay  dynamicconfig.DurationPropertyFn
 	StandbyTaskMissingEventsDiscardDelay dynamicconfig.DurationPropertyFn
 
+	QueueCriticalReadWatermarkAttempts dynamicconfig.IntPropertyFn
+	QueueMaxReaderCount                dynamicconfig.IntPropertyFn
```

Comment:
I can create separate configs for each queue type but for now I don't think that's necessary?